src/constants/vuetorrent/Comparators.ts

```diff
@@ -33,8 +33,16 @@ const comparatorMap: Record<keyof Torrent, (a: Torrent, b: Torrent, isAsc: boole
   last_activity: (a, b, isAsc) => comparators.invertedNumeric.compare(a.last_activity, b.last_activity, isAsc),
   magnet: (a, b, isAsc) => comparators.text.compare(a.magnet, b.magnet, isAsc),
   name: (a, b, isAsc) => comparators.textWithNumbers.compare(a.name, b.name, isAsc),
-  num_leechs: (a, b, isAsc) => comparators.numeric.compare(a.num_leechs, b.num_leechs, isAsc),
-  num_seeds: (a, b, isAsc) => comparators.numeric.compare(a.num_seeds, b.num_seeds, isAsc),
+  num_leechs: (a, b, isAsc) => {
+    const primaryComparison = comparators.numeric.compare(a.num_leechs, b.num_leechs, isAsc)
+    if (primaryComparison === 0) return comparatorMap['available_peers'](a, b, isAsc)
+    else return primaryComparison
+  },
+  num_seeds: (a, b, isAsc) => {
+    const primaryComparison = comparators.numeric.compare(a.num_seeds, b.num_seeds, isAsc)
+    if (primaryComparison === 0) return comparatorMap['available_seeds'](a, b, isAsc)
+    else return primaryComparison
```

Comment:
Useless else?

Reply:
Yup. I was just following the style of the sort key after this one which also has a redundant else:
https://github.com/VueTorrent/VueTorrent/pull/2021/files#diff-2731caa080eabd74ef3cbcccb750ede8eba9d53e9439c5e915ab13e8e55ce446R50

If I don't hear back from you I'll remove it as suggested when I come back to this PR to make these changes.

Reply:
For conditional return with only 2 conditions, I would personally use a terminary operator instead but an if-else is still acceptable.

The case you mentioned is a bit particular because it's a 4-case condition.

Anyway, it gets compiled to ternaries anyway by vite.